tools.sh

```
@@ -219,7 +219,11 @@ function create_vm {
 function generate_htpasswd_file {
    local auth_file_dir=$1
    local pass_file=$2
-   random_password=$(cat $1/auth/kubeadmin-password)
-   ${HTPASSWD} -c -B -b ${pass_file} developer developer
-   ${HTPASSWD} -B -b ${pass_file} kubeadmin ${random_password}
+   (
+       set +x # use a subshell to avoid leaking the password
+
+       local random_password=$(cat $1/auth/kubeadmin-password)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)